scripts/build_osm_network.py

```diff
@@ -908,12 +911,33 @@ def built_network(
 
     sets_path_to_root("pypsa-earth")
 
+    # Keep only a predefined set of columns, as otherwise conflicts are possible
+    # e.g. the columns which names starts with "bus" are mixed up with
+    # the third-bus specification when executing additional_linkports()
+    lines_cols_standard = [
```

Comment:
This could also be a global variable at the beginning with capitalized characters.

Reply:
> Done. Have I got your idea?

Great, almost; would you mind placing it around line 25-26ish?
Having it as a global variable could help as it may be loaded by other scripts if it will ever be necessary.

Reply:
Makes sense :) Done!